packages/xml-views-validation/test/validators/element/unknown-tag-name-spec.ts

```diff
@@ -310,6 +310,18 @@ describe("the unknown tag name validation", () => {
             </mvc:View>`
           );
         });
+
+        it("will not detect an issue for a class under an none existent namespace which is also a valid UI5 entity FQN", () => {
+          assertNoIssues(
+            `<mvc:View
+              xmlns:mvc="sap.ui.core.mvc"
+              <!-- The error should be on the definition of an invalid xmlns which is not a UI5Namespace rather than on each usage -->
+              xmlns:AvatarColor="sap.m.AvatarColor">
+              <AvatarColor:Green>
```

Comment:
Why shouldn't this give an error? `Green` isn't a class (or aggregation), it can't appear in a tag name. An error here is not a false positive. It would fail in runtime.

Reply:
If we had `xmlns:bamba="sap.m.Bamba" would we have given an error?`

Reply:
Also there is only one error here imho, the invalid namespace.
Every usage of the namespace in a tag should not be an error.

Reply:
> If we had `xmlns:bamba="sap.m.Bamba" would we have given an error?`

We would have given a warning on the namespace because we don't know it and it's in an sap library namespace (sap.m). We would not have given an error (or done validations, or code assist) on tags that use this namespace because we consider it to be a custom namespace.

This isn't the case when you put an existing symbol name in the xmlns attribute. That **is** known (so we can't say it's an "unknown namespace"). It cannot be considered a custom namespace (so we cannot treat tags that use it as possible custom controls that should be excluded from validations). And it might have classes inside it which we should validate and suggest for code completion.